pipelines/epinow2/generate_rerun_config.py

```diff
@@ -47,19 +48,25 @@
     # so that is only handled in this script.
     excl_path: str | None = os.getenv("data_exclusions_path")
     if excl_path is None:
-        raise ValueError("data_exclusions_path not found in environment variables.")
+        # Set the default to the standard production location, with this report date.
+        # Make sure we have a properly formatted date.
+        rd: date = (
+            date.fromisoformat(user_args["report_date"])
+            if isinstance(user_args["report_date"], str)
+            else user_args["report_date"]
+        )
+        excl_path = f"az://nssp-rt-v2/outliers/{rd.isoformat()}.csv"
```

Comment:
Can we move this container name to a constant? Maybe here: https://github.com/CDCgov/cfa-config-generator/blob/main/src/cfa_config_generator/utils/epinow2/constants.py#L78

Reply:
added